### PR TITLE
Repair the tests via supplying test cases and adding missing keywords.json file

### PR DESCRIPTION
**Pull request by matthewrkarlsen**
Modifications:

1. Update `.gitignore` to include IntelliJ `.idea` folder
2. Pull in test case JSON files via `maven-scm-plugin`
3. Supply `keywords.json` file to be read in via `org.jsonrql.Keywords`
4. Update `QueryTest` to use the new JSON test cases location

Open questions:

1. is the current path to which the test cases are pulled in ok?
2. is the `QueryTest.java` file as you would want it?

### CODE REVIEW

File: src/main/java/org/jsonrql/Keywords.java [hunk 1]
```diff
@@ -22,7 +23,8 @@
     {
         try
         {
-            KEYWORDS = new ObjectMapper().readValue(Keywords.class.getResource("keywords.json"), Keywords.class);
+            URL resource = Keywords.class.getClassLoader().getResource("keywords.json");
```
Comment (gsvarovsky) [inline src/main/java/org/jsonrql/Keywords.java]: personally I always use `final` if the variable won't change. But gosh it feels like a long time ago I was doing Java. Your thoughts welcome if you disagree!

File: src/test/java/org/jsonrql/QueryTest.java [hunk 2]
```diff
@@ -38,9 +42,13 @@ private static Stream<URL> testCases() throws IOException
...
-                                                     System.getProperty("json-rql.version"),
-                                                     tc)));
+        return testNames.stream()
+                .map(tc -> {
+                    try {
+                        return new File(format("target/test-cases/test/data/%s.json", tc)).toURI().toURL();
+                    } catch (MalformedURLException e) {
+                        throw new RuntimeException(e);
+                    }
+                });
```
Comment (gsvarovsky) [inline src/test/java/org/jsonrql/QueryTest.java:52]: Choice of test case location, and code totally fine & great 💯

**Review by gsvarovsky — APPROVED** [review]
Super, thanks Matthew! Tiny coding standard nitpick added.

**Comment by matthewrkarlsen** [comment]
Hello @gsvarovsky, have made the requested modification.

Would you like me to merge or squash and merge, please?

**Comment by gsvarovsky** [comment]
Entirely up to you. Nice job!

**Pull request merged**